dotCMS/src/main/java/com/dotmarketing/db/DotCMSInitDb.java

```diff
@@ -113,11 +113,11 @@ private static void loadStarterSite(PrintWriter pw) throws IOException{
 			String zipPath = FileUtil.getRealPath(starterSitePath);
 			starterZip = new File(zipPath); 
 		 }
-		
-		ImportExportUtil ieu = new ImportExportUtil();
-		if(ieu.validateZipFile(starterZip)){
-		    Try.run((()->ieu.doImport(pw))).onFailure(e-> new DotRuntimeException(e));
 
-		}
+		ImportExportUtil ieu = new ImportExportUtil();
+		if(ieu.validateZipFile(starterZip))
+			Try.run((() -> ieu.doImport(pw))).onFailure(e -> {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This statement should have braces](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4610406)